core/domain/exp_domain.py

```diff
@@ -39,8 +39,11 @@
 from core.domain import state_domain
 from core.domain import translation_domain
 
-from typing import Dict, List, Optional, Tuple
-from typing_extensions import TypedDict
+from typing import (
+    Any, Callable, Dict, List, Mapping, Optional, Sequence,
```

Comment:
We shouldn't be using Any.

Reply:
This Any is used in only one place which is:
https://github.com/oppia/oppia/blob/a6f1a20b482e41083cc223f4dafde60bdf7c27df/core/domain/exp_domain.py#L2087
PTAL at the comment and docstring.